src/js/menuItem.js

```diff
@@ -75,13 +75,13 @@ export function renderSpecialItems() {
     special.innerHTML = '';
 
     menuData.forEach(item => {
-        if (item.tags.includes("special!")) {
+        if (item.tagArray.includes("special!")) {
             const specialCard = document.createElement('article');
             specialCard.classList.add('menu-product-card');
             specialCard.innerHTML = `
```

Comment:
Net zoals eerder, liever geen `innerHTML` gebruiken: https://www.geeksforgeeks.org/javascript/what-is-the-disadvantage-of-using-innerhtml-in-javascript/